src/vstest.console/Processors/Utilities/StringUtilities.cs

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.VisualStudio.TestPlatform.CommandLine.Processors.Utilities
+{
+    using System.Collections.Generic;
+    using System.Text;
+
+    internal static class StringExtensions
+    {
+        public static IEnumerable<string> Tokenize(this string input, char separator, char escape)
```

Comment:
Please add test cases for this class.

Check if input = "\" or "\\" or "\,"

Reply:
Or make this a `private static` method until we need another consumer for this.

Reply:
Please do not make it private, once you do it, it will forever remain in this dll :(. Please move it to Utilities dll, under String Utilities class

Reply:
Done